app/src/main/java/com/example/concordia_campus_guide/Adapters/PlaceToSearchResultAdapter.java

```diff
@@ -0,0 +1,120 @@
+package com.example.concordia_campus_guide.Adapters;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ArrayAdapter;
+import android.widget.Filter;
+import android.widget.Filterable;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+
+import com.example.concordia_campus_guide.Models.Place;
+import com.example.concordia_campus_guide.R;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PlaceToSearchResultAdapter extends ArrayAdapter<Place> implements Filterable {
+
+    Context c;
+    List<Place> places;
+    List<Place> originalPlaces;
+    CustomFilter cs;
+
+    private static class ViewHolder {
+        private TextView itemView;
+    }
+
+    public PlaceToSearchResultAdapter(Context context, int textViewResourceId, List<Place> items) {
+        super(context, textViewResourceId, items);
+        c = context;
+        places = items;
+        originalPlaces = items;
+    }
+
+    @Override
+    public View getView(int position, View convertView, ViewGroup parent) {
+        final ViewHolder viewHolder;
+
+        if (convertView == null) {
+            convertView = LayoutInflater.from(this.getContext())
+                    .inflate(R.layout.list_item_layout, parent, false);
+
+            viewHolder = new ViewHolder();
+            viewHolder.itemView = (TextView) convertView.findViewById(R.id.textView);
+
+            convertView.setTag(viewHolder);
+
+        } else {
+            viewHolder = (ViewHolder) convertView.getTag();
+        }
+
+        Place item = getItem(position);
+        if (item!= null) {
+            // My layout has only one TextView
+            // do whatever you want with your string
```

Comment:
do we need to keep these comments ?

Reply:
lol thanks for catching this

Reply:
actually, I would leave it because this will be useful for UI @ferdousara-parvin

Reply:
Can you make it as Todo ??